homework-04/models/mongo/lastModifiedPlugin.js

```diff
@@ -0,0 +1,14 @@
+function lastModifiedPlugin (schema, options) {
+  schema.add({ lastModifiedDate: Date })
+
+  schema.pre('save', function (next) {
```

Comment:
Please, do the same thing on schema update

Reply:
Done